packages/shared/src/hooks/useChat.ts

```diff
@@ -65,106 +63,13 @@ interface TokenPayload {
   token: string;
 }
 
-export const useChat = ({ id: idFromProps, query }: UseChatProps): UseChat => {
+export const useChatStream = (): Pick<UseChat, 'handleSubmit'> & {
```

Comment:
Small request can we maybe make a `chat` or `useChat` directory inside hooks which exports a index and split these three in seperate files?
I'm more a fan of keeping files smaller and easier to distinguish.

Reply:
For sure 👍